Wikipedia/Code/WMFAppViewController.m

```diff
@@ -901,7 +901,9 @@ - (void)finishResumingApp {
                                                    completion:^{
                                                    }];
     } else if (locationAuthorized != [defaults wmf_locationAuthorized]) {
-        [self.dataStore.feedContentController updateNearbyForce:NO completion:NULL];
+        [self.dataStore.feedContentController updateContentSource:[WMFNearbyContentSource class] force:NO completion:NULL];
+    } else if (!NSUserDefaults.wmf.shouldShowLastReadArticleOnResume) {
```

Comment:
Seems like this should be a separate `if` instead of an `else if` since it should be able to happen whether or not nearby was updated due to a location authorization change